pkgs/build-support/docker/default.nix

```diff
@@ -316,13 +316,20 @@ rec {
     # Files to add to the layer.
     contents,
     baseJson,
+    extraCommands,
     uid ? 0, gid ? 0,
   }:
     runCommand "${name}-customisation-layer" {
       buildInputs = [ jshon rsync tarsum ];
+      inherit extraCommands;
     }
     ''
       cp -r ${contents}/ ./layer
+      chmod ug+w layer
```

Comment:
Is this needed when we don't have any `extraCommands`?

Reply:
good point, fixed